ports/servoshell/egl/ohos/simpleservo.rs

```diff
@@ -34,7 +34,7 @@ pub fn init(
     xcomponent: *mut OH_NativeXComponent,
     waker: Box<dyn EventLoopWaker>,
     callbacks: Box<dyn HostTrait>,
-) -> Result<ServoGlue, &'static str> {
+) -> Result<Rc<RunningAppState>, &'static str> {
```

Comment:
Is there a Reason we wrap this in an `Rc` now? The ohos port doesn't seem to require this.

Reply:
The `RunningAppState` implements the `WebViewDelegate` and `ServoDelegate` traits which require `Rc<...>` so that they can be shared between `WebView`s.

Reply:
Here's an example of the delegate pattern in the WebKit API: https://developer.apple.com/documentation/webkit/wkwebview/uidelegate?language=objc

Reply:
Ah, yes. That is true for servoshell, but for the delegate itself the default behavior is to deny.